lib/utils.js

```diff
@@ -27,25 +28,27 @@ async function getListOfFilesToReplicate(octokit, commitId, owner, repo, filesTo
   }
 
   if (triggerEventName === 'workflow_dispatch') {
-    const workflowDirPath = path.join(process.cwd(), defaultWorkflowsDir);
-    const workflowDirFilesList = await readdir(workflowDirPath);
-    filesToCheckForReplication = workflowDirFilesList.map(filename => path.join(defaultWorkflowsDir, filename));
-    core.debug(`DEBUG: list of files from ${workflowDirPath} directory is ${filesToCheckForReplication}`);
+    const workspaceDirPath = path.join(process.cwd());
+    const workspaceDirFilesList = await getAllFiles(workspaceDirPath);
+    filesToCheckForReplication = workspaceDirFilesList.map(filename => path.join(filename.replace(workspaceDirPath, '').substring(1)));
```

Comment:
can you add a comment on what is actually happening here, we are we doing the replacement, 

also I guess `path.join` is no longer needed

Reply:
You ever go back to code you wrote a long time ago and wonder what on earth you were thinking, this is one of those situations.